### ISSUE TITLE
Is there documentation of what each method and class does? / Do you need someone to produce this documentation

### ISSUE THREAD

**Issue by humanitiesclinic** [farafiri/PHP-parsing-tool#39]
Is there documentation of what each method and class does? Do you need someone to produce this documentation, if there isn’t such?

**Comment by farafiri** [comment]
No, there is no such a documentation. I must say I never thought about that as I an never using such things.

**Comment by farafiri** [comment]
Do You feel like there is a need for such thing?

**Comment by humanitiesclinic** [comment]
I think it would definitely be of great help to those who are not the creator. It will fast track our comprehension of the project especially if it has many classes like yours... Do consider.
